ext/panko_serializer/attributes_iterator.c

```diff
@@ -1,24 +1,27 @@
 #include "attributes_iterator.h"
 
-static ID attributes_id;
-static ID types_id;
-static ID values_id;
-static ID delegate_hash_id;
 
-static ID value_before_type_cast_id;
-static ID type_id;
+ID attributes_id;
+ID types_id;
+ID values_id;
+ID delegate_hash_id;
+
```

Comment:
What was the reason for making these non-static? I don't see them anywhere else in the code other than in this file.

Reply:
My bad, forgot to fix this.
Will make them static again and use `rb_gc_register_address`

Reply:
I think you pointed out that ID don't get registered. Changing from non-static to static should not make any difference in regards to GC.

Reply:
@ohler55 I am not sure about this area at all.
I saw in Oj, you don't rb_gc_register_address for ID, and as far as I know, ID isn't GCed.

I will try to find some resources about it and see if `rb_gc_register_address` is needed here

Reply:
I don't think it is needed.

Reply:
cool!